.teamcity/settings/extensions.kt

```diff
@@ -19,3 +19,5 @@ fun BuildType.gradleProfilerVcs() {
         checkoutMode = CheckoutMode.ON_SERVER
     }
 }
+
+val useBuildScanInitScript = "-I gradle/init-scripts/build-scan.init.gradle.kts"
```

Comment:
This is not really about using build scans, but about configuring our own server, no? I mean, the `build-scan` plugin would be applied anyway from `build.gradle.kts`, no?

Reply:
Do you suggest to rename the `build-scan.init.gradle.kts` script? Or is this more about renaming `useBuildScanInitScript`? I think the variable `useBuildScanInitScript` is fine, since the command line argument instructs Gradle to use the `build-scan` init script.

Reply:
How about something like `useGradleInternalScansServer` and `build-scan-gradle-internal-server.init.gradle.kts`?

Reply:
I renamed the variable and the script.